mappings/net/minecraft/enchantment/Enchantment.mapping

```diff
@@ -1,17 +1,53 @@
 CLASS none/ahm net/minecraft/enchantment/Enchantment
+	CLASS none/ahm$a Weight
+		FIELD a COMMON Lnone/ahm$a;
+		FIELD b UNCOMMON Lnone/ahm$a;
+		FIELD c RARE Lnone/ahm$a;
+		FIELD d LEGENDARY Lnone/ahm$a;
```

Comment:
Maybe the weights should be named in a more standard way like: EXTREME, HIGH, MEDIUM, LOW.